source-map.bs

```diff
@@ -577,10 +577,10 @@ To <dfn>decode a base64 VLQ</dfn> from a [=string=] |segment| given a [=position
 NOTE: In addition to returning the decoded value, this algorithm updates the [=position variable=]
 in the calling algorithm.
 
-### [=Mappings=] for generated JavaScript code
+### [=json/Mappings=] for generated JavaScript code ### {#mapping-javascript}
```

Comment:
Shouldn't these point to the "decoded mappings struct" because we talk about "mapping entries"? In the "names" PR I changed these to `[=decoded mapping|Mapping=]`.

Reply:
Thanks, that sounds good to me. I revised this PR to match the names PR (#122) so there's no conflict here.